<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Database Darkly - Database Taxa entries</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="solar.scss">
</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Database Darkly</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">Main</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./how-to.html" rel="" target="">
 <span class="menu-text">How to use</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./db-main.html" rel="" target="">
 <span class="menu-text">Database curation</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./db-taxa.html" rel="" target="" aria-current="page">
 <span class="menu-text">Summary of taxa</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#supergroup-alveolata" id="toc-supergroup-alveolata" class="nav-link active" data-scroll-target="#supergroup-alveolata"><span class="header-section-number">1</span> Supergroup: Alveolata</a>
  <ul class="collapse">
  <li><a href="#phylum-dinoflagellates" id="toc-phylum-dinoflagellates" class="nav-link" data-scroll-target="#phylum-dinoflagellates"><span class="header-section-number">1.1</span> Phylum: Dinoflagellates</a></li>
  <li><a href="#phylum-ciliates" id="toc-phylum-ciliates" class="nav-link" data-scroll-target="#phylum-ciliates"><span class="header-section-number">1.2</span> Phylum: Ciliates</a></li>
  <li><a href="#phylum-apicomplexa" id="toc-phylum-apicomplexa" class="nav-link" data-scroll-target="#phylum-apicomplexa"><span class="header-section-number">1.3</span> Phylum: Apicomplexa</a></li>
  <li><a href="#phyum-perkinsea" id="toc-phyum-perkinsea" class="nav-link" data-scroll-target="#phyum-perkinsea"><span class="header-section-number">1.4</span> Phyum: Perkinsea</a></li>
  </ul></li>
  <li><a href="#supergroup-amoebozoa" id="toc-supergroup-amoebozoa" class="nav-link" data-scroll-target="#supergroup-amoebozoa"><span class="header-section-number">2</span> Supergroup: Amoebozoa</a>
  <ul class="collapse">
  <li><a href="#phylum-conosa" id="toc-phylum-conosa" class="nav-link" data-scroll-target="#phylum-conosa"><span class="header-section-number">2.1</span> Phylum: Conosa</a></li>
  </ul></li>
  <li><a href="#phylum-apusozoa" id="toc-phylum-apusozoa" class="nav-link" data-scroll-target="#phylum-apusozoa"><span class="header-section-number">3</span> Phylum: Apusozoa</a></li>
  <li><a href="#supergroup-archaeplastida" id="toc-supergroup-archaeplastida" class="nav-link" data-scroll-target="#supergroup-archaeplastida"><span class="header-section-number">4</span> Supergroup: Archaeplastida</a></li>
  <li><a href="#supergroup-excavata" id="toc-supergroup-excavata" class="nav-link" data-scroll-target="#supergroup-excavata"><span class="header-section-number">5</span> Supergroup: Excavata</a></li>
  <li><a href="#supergroup-hacrobia" id="toc-supergroup-hacrobia" class="nav-link" data-scroll-target="#supergroup-hacrobia"><span class="header-section-number">6</span> Supergroup: Hacrobia</a></li>
  <li><a href="#supergroup-opisthokonta" id="toc-supergroup-opisthokonta" class="nav-link" data-scroll-target="#supergroup-opisthokonta"><span class="header-section-number">7</span> Supergroup: Opisthokonta</a></li>
  <li><a href="#supergroup-rhizaria" id="toc-supergroup-rhizaria" class="nav-link" data-scroll-target="#supergroup-rhizaria"><span class="header-section-number">8</span> Supergroup: Rhizaria</a></li>
  <li><a href="#supergorup-stramenopiles" id="toc-supergorup-stramenopiles" class="nav-link" data-scroll-target="#supergorup-stramenopiles"><span class="header-section-number">9</span> Supergorup: Stramenopiles</a>
  <ul class="collapse">
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">9.0.1</span> <strong>References</strong></a></li>
  </ul></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Database Taxa entries</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="supergroup-alveolata" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Supergroup: Alveolata</h1>
<p>Alveolata (or Alveolates) contains well-known groups of protists: Apicomplexa, Ciliophora, and Dinoflagellata. Grouped with Stramenopiles and Rhizaria, the three clades also form what is known as the SAR supergroup. Alveolata share the presence of membrane-enclosed sacs, or vesicles, just beneath the cell membrane. In Dinoflagellata (dinoflagellates), these sacs can contain plates, forming a sort of armor. The sacs ensure a thick yet flexible cell wall in naked cells. Alveolata also share tubulocristate mitochondria, characteristic of the SAR group. Alveolata are ubiquitous in deep-sea environments and often make up the most diverse group present. For instance, MArine ALveolates (or MALV) is an order of parasitic dinoflagellates that are frequently found in deep-sea samples.</p>
<section id="phylum-dinoflagellates" class="level2" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="phylum-dinoflagellates"><span class="header-section-number">1.1</span> Phylum: Dinoflagellates</h2>
<p>The Phylum Dinoflagellata contains around 2,400 living species and even more fossilized species. Dinoflagellates are biflagellated, possessing a transverse and longitudinal flagellum which differ functionally and morphologically, and have retained alveoli. They vary in form and function, being thecate or athecate, but generally pear-shaped, photosynthetic or non-photosynthetic, and free-living or parasitic. Dinoflagellates are abundant in both freshwater and marine environments from the equator to the poles, and even in snow and sea ice. They are second, only to diatoms in oceanic primary production with half of the species being photosynthetic. Dinoflagellates employ a variety of trophic strategies including mixotrophy, phagotrophy, osmotrophy, and myzocytosis. Photosynthetic dinoflagellates are typically mixotrophic, involving heterotrophy, as well. Many species are symbiotic as mutualists and parasites. The most notable mutualistic dinoflagellates are zooxanthellae, having relationships with coral and other invertebrates. The order Syndiniales, also known as Marine Alveolates, are parasitic dinoflagellates whose hosts include crustaceans, fish, cnidarians, and protists, including other dinoflagellates. The Marine Aleveolates consist of two groups whose sequences have frequently been obtained in deep-sea eukaryotic diversity studies. There are many toxic species, several of which can form blooms that kill or cause illness to other organisms. The function of the toxins in other dinoflagellates is understudied, as they do not seem to deter predators. Benthic dinoflagellates and their effect on marine ecosystems are greatly understudied, despite their presence being common in sediments. Notable features of dinoflagellates include a circadian rhythm and bioluminescence in some species.</p>
</section>
<section id="phylum-ciliates" class="level2" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="phylum-ciliates"><span class="header-section-number">1.2</span> Phylum: Ciliates</h2>
<p>The Phylum Ciliophora is a large group of protists, almost all of which are free-living heterotrophs. Ciliates are distinguished by three key traits: a ciliated cell surface, at some stage of life; nuclear dimorphism, the presence of two morphologically and functionally distinct nuclei, the macronucleus and micronucleus; and conjugation, a sexual process in which compatible mates temporarily fuse to exchange genetic material of the micronuclei. The eponymous cilia are typically used for locomotion and feeding, the modes of which are relatively diverse. Ciliates feed upon bacteria, smaller protists, such as algae, and other ciliates. Some ciliates retain ingested bacteria, such as methane and hydrogen sulfide-utilizing bacteria, which continue to function, providing nutrients and space for continued growth and production. This extends to the chloroplasts of ingested algae which can be retained rather than digested, enabling primary production in the ciliate, a process referred to as kleptoplasty. Ciliates can also be symbionts in other organisms, such as ruminal ciliates which aid in the digestion processes of their hosts. There are over 8,000 species of ciliates, distributed globally, in a wide variety of environments, occupying an important trophic level in these ecosystems.</p>
</section>
<section id="phylum-apicomplexa" class="level2" data-number="1.3">
<h2 data-number="1.3" class="anchored" data-anchor-id="phylum-apicomplexa"><span class="header-section-number">1.3</span> Phylum: Apicomplexa</h2>
<p>Apicomplexans are a large phylum of unicellular, parasitic protists that range from a symmetrical round to an elongated shape. They utilize myzocytotic ingestion, also known as ‘cellular vampirism’, in which they pierce the cell membrane of their prey with a feeding tube and proceed to suck out the contents and digest them. This is a method of passive ambush feeding, where they wait for prey and then secure them through the use of a quick ambush.</p>
<p>Through their parasitism, apicomplexans can pose a great risk to the health of the host organism. They are able to conduct these parasitic invasions through the use of a collection of organelles termed the apical complex. These organelles, particularly rhoptries and micronemes, are what gives them the ability to adhere to host cells, subsequently invade the host cells, and establish a parasitophorous vacuole, which is what surrounds the parasite and protects it from the host’s cellular defenses.</p>
<p>The most interesting feature of the apicomplexans is their possession of an apicoplast, a relict photosynthetic plastid that is only found in this phylum. While the plastids found in photosynthetic organisms, such as plants and algae, are largely known for their green appearance and being the location of photosynthesis, they do not possess either of those traits as they exist in apicomplexans. These organelles are essential to apicomplexan’s survival, as without them the organism would perish.</p>
</section>
<section id="phyum-perkinsea" class="level2" data-number="1.4">
<h2 data-number="1.4" class="anchored" data-anchor-id="phyum-perkinsea"><span class="header-section-number">1.4</span> Phyum: Perkinsea</h2>
<p>The Perkinsea phylum is a group of protists consisting of the marine genera Perkinsus and Parvilucifera and the freshwater genera Phagodinium and Rastrimonas. This phylum is mainly parasitic and is the closest sister group to dinoflagellates. These parasitic marine protists have been frequently detected to thrive in the sediments and seawater of hydrothermal vents and have been thought to negatively affect the community structure and population of the organisms inhabiting these sites. Perkinsids largely infect bivalves, with the exception of the genus Parvilucifera which infects other dinoflagellates. Because of this, perkinsid infestation has devastated the aquaculture of mollusks and has the potential to cause harmful algal blooms.</p>
<p>These protists have a naked cell cover, a round shape, and an attached motility. They also have a spherical symmetry, are heteropolar, and are ectosymbiontic. Except for Parvilucifera, which has a bilateral symmetry and is endosymbiontic. Perkinsids are also, on average, passive ambush feeders with myzocytotic ingestion. Perkinsids possess the “archetype” apical complex and utilize it for the invasion of their host. In perkinsids, this apical complex extends from the flagellar apparatus and consists of an open-sided conoid and diverse vesicular components (such as micronemes). After a successful invasion, Perkinsus marinus develops a trophozoite with a spherical nucleus and a vacuole that occupies the majority of the cytoplasm space. The trophozoite then undergoes palintomy in the vegetative state, wrestling in the schizont containing 4-64 immature trophozoites. After rupturing of the schizont’s cell wall, the immature trophozoites spread throughout the host cell body and asexual proliferation continues. As the host dies and the tissue becomes anoxic, the trophozoite enlarges and becomes a pre-zoosporangium (hypnospore) with a thicker cell wall. However, in marine environments, the pre-zoosporangium develops into a zoosporangium with a discharge tube. The zoosporangium undergoes palintomy, producing numerous biflagellated zoospores, which may serve as an infective stage. Alternatively, after Parvilucifera sporangia’s successful invasion, they will grow within the cytoplasm until they occupy nearly the entire host cell. Once mature, a germ tube then forms and releases zoospores.</p>
<p>The Perkinsea phylum and its relation to dinoflagellates and apicomplexans is extremely important for our understanding of early evolution. However, there is not enough research on perkinsids for this evolution to be fully understood. Recent data suggests that this phylum is much more diverse than previously thought and further investigation could reveal critical information regarding the connection of lineages.</p>
</section>
</section>
<section id="supergroup-amoebozoa" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Supergroup: Amoebozoa</h1>
<p>Amoebozoa encompass a wide variety of amoeboid organisms possessing pseudopodia, with no defined shape, and can be free-living or parasitic. The pseudopodia are used to anchor the organism to a substrate, for locomotion, and feeding. Amoebozoans can also propel themselves by flowing their cytoplasm in a forward direction. The cell cover of amoebas can be naked, with or without a coating of glycolax, testate, or scaled.</p>
<p>The group is divided into two subphylums: Lobosa and Conosa. Lobose amoebae are characterized by lobe-like pseudopodia while Conosa have thread-like pseudopodia and are further distinguished by a flagellate stage in their lifecycle. The vast majority of Amoebazoans are unicellular; however, Mycetozoans, or slime molds, have a multicellular life stage in which they form macroscopic aggregates.</p>
<p>The Supergroup is entirely heterotrophic, employing phagocytosis as the primary mode of ingestion, and typically predating on bacteria, algae, small protists, and yeast. In unfavorable conditions, such as food scarcity, amoebas can encyst, and the excyst when conditions improve. Amoebozoa are present in deep-sea environments, but their abundance and diversity within these communities are vastly understudied. Furthermore, the phylogeny and taxonomy of Amoebozoa remain poorly understood and controversial.</p>
<section id="phylum-conosa" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="phylum-conosa"><span class="header-section-number">2.1</span> Phylum: Conosa</h2>
<p>Conosa is one of the phyla included in the Amoebozoa supergroup. They exist in a wide range of habitats, from aquatic to terrestrial, and can even survive in anoxic soil environments. The majority of this phylum have a naked cell cover, are phagotrophic, and utilize an ambush feeding tactic. An amoeboid shape is common, and they have a motility style ranging from gliding to swimming or floating. Conosa can be identified morphologically by a monolayer of microtubules that either partially or completely surround the anterior basal body. This can then either diverge towards the nucleus and cell posterior as a half or three-quarters open cone (as seen in Variosea and Mycetozoa) or as a complete cone (as seen in Archaemoebae). The flagellar apparatus is an ancestral trait and has been evolutionally changed to suit the needs of different lineages. For example, for Pelobiontida, the posterior flagellum and associated cytoskeleton have been lost and for Entamoebidae the whole flagellar apparatus has been lost.</p>
</section>
</section>
<section id="phylum-apusozoa" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Phylum: Apusozoa</h1>
<p>Apusomonads are morphologically classified as having two flagella at the anterior of the cell. They are motile, via a gliding motion. The phylogenetic placement of this group continue to be debated, but is more often labeled as a phylum (within the Obazoa) that contains the groups Breviata and Apusomonadida (and sometimes Mantamonadida). In the deep sea, Apusozoa are commonly found (but not in high cell abundance) within niche habitats, including sediments and hydrothermal vent fluid. All species within Apusozoa are phagotrophic and are able to actively hunt prey, mainly consisting of bacteria.</p>
</section>
<section id="supergroup-archaeplastida" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Supergroup: Archaeplastida</h1>
<p>The Supergroup Archaeplastida has chloroplast encompassed by two membranes, thought to have been obtained via primary endosymbiosis with a cyanobacterium. Evidence suggests that this plastid originated from a singular event, implying that a common ancestor of Archaeplastida engulfed a cyanobacterium which was retained and passed onto offspring. The supergroup is divided into 3 subgroups: Glaucophyta, Rhodophyta, and Chloroplastida. Chloroplastids, further divided into phylums Streptophyta and Chlorophyta, contain green algae and land plants. Rhodophytes, or red algae, contain marine algae, most of which are seaweed. These two phyla feature a diverse array of cellularity from simple unicellular, flagellated or non-flagellated, colonial, filamentous, or multicellular and macroscopic. The third subgroup is the Glaucophytes, rare freshwater algae associated with surface water. Most Archaeplastids are photosynthetic, but parasitic taxa exist within red algae and land plants, and the few that are heterotrophic employ osmotrophy as their mode of ingestion. Their presence in the deep sea is scarce compared to other microeukaryotic supergroups, and the sequences that have been observed in the deep sea are hypothesized to be from sinking matter.</p>
</section>
<section id="supergroup-excavata" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Supergroup: Excavata</h1>
<p>The Excavata supergroup is a diverse group of organisms that include heterotrophic predators, photosynthetic species, and parasites. This group of protists is of particular interest as it has been largely absent in the global ocean, but has rather been located within the hydrothermal vent fluid range. These organisms tend to be asymmetrical, single-celled with a feeding groove on the side of their bodies, leading to the term “excavated” which serves as the root of their name. One interesting feature is that many species of this supergroup lack the ‘classical’ mitochondria that many other eukaryotic cells possess, but have an organelle referred to as ‘amitochondriates’. However, many of them still contain some kind of greatly modified mitochondrial organelle.</p>
<p>Metamonada have adapted metabolisms to low oxygen conditions, largely due to the lack of an aerobic mitochondria. This is one reason as to how this supergroup is able to thrive in large numbers in hydrothermal vent sites, as many phyla are adapted to live in anoxic environments.</p>
</section>
<section id="supergroup-hacrobia" class="level1" data-number="6">
<h1 data-number="6"><span class="header-section-number">6</span> Supergroup: Hacrobia</h1>
<p>The supergroup Hacrobia is a controversial topic among researchers. There are many debates and conflicting classifications of this supergroup. It has several names, the most common being Hacrobia, but occasionally being referred to as CCTH (Cryptophyta, Centrohelida, Telonemia, Haptophyta) or Eukaryomonadae. Hacrobia is also classified as a subkingdom under Chromista. This supergroup resides in a wide variety of environments ranging from marine or terrestrial to freshwater or brackish. Currently, Hacrobia is classified as a photosynthetic clade, but phylogenetically it is being debated whether to classify it as monophyletic or polyphyletic. Haptophyta was previously considered a sister group to the SAR group and Cryptophyta was previously a cluster with Archaeplastida.</p>
<p>The majority of protists within Hacrobia are eaten by dinoflagellates, others by ciliates, and a few are consumed by kathablepharids. Hacrobia is classified as having a naked or organic shape and being heteropolar with a swimming motility. Many are haplodiplontic, meaning there are two multicellular stages (haploid and diploid), and are ambush feeders. The controversy surrounding research on this group has led to conflicting reports about Hacrobia’s ecology. One paper discussed how Hacrobians were absent at higher salinity levels, but another described that they were much more abundant in euphotic zones rather than the deep sea.</p>
<p>This Supergroup and its classification continue to be a hotly debated topic throughout scientific communities. Through more research, our understanding of Hacrobia will continue to provide essential information about hydrothermal vent ecology.</p>
</section>
<section id="supergroup-opisthokonta" class="level1" data-number="7">
<h1 data-number="7"><span class="header-section-number">7</span> Supergroup: Opisthokonta</h1>
<p>The Supergroup Opisthokonta includes a diverse array of organisms including fungi, choanoflagellates, and animals. Opisthokonts are characterized by a singular posterior flagellum at some point in their life. The supergroup is divided into two subclades: Holomycota (nucleariids, microsporidia, and fungi) and Holozoa (Icthyosporea, Filasterea, Choanozoa, and Metazoa). Opisthokonts are exclusively heterotrophic and are typically phagotrophic or osmotrophic, inhabiting both terrestrial and aquatic environments. There are no species that contain chloroplasts although some hold symbiotic or kleptoplastidic relationships with algae. Opisthokonts play a large role in oceanic nutrient cycling. Fungi are known for their role as decomposers in terrestrial ecosystems, yet their role in marine ecosystem nutrient cycling remains understudied, while Metazoans are well-studied, major consumers in deep-sea ecosystems. Non-protist opisthokonts are relatively less common in extreme, deep-sea environments, but some species have adapted to the harsh conditions. Furthermore, Metazoans also hold importance as hosts for a variety of other organisms in deep-sea environments.</p>
</section>
<section id="supergroup-rhizaria" class="level1" data-number="8">
<h1 data-number="8"><span class="header-section-number">8</span> Supergroup: Rhizaria</h1>
<p>The Supergroup Rhizaria, the third major group within the SAR clade, contains well-known phyla including Radiolarians, Foraminiferans, and Cercozoans. Rhizarians are characterized by pseudopodia, often thread or needle-like, with cytoplasmic strands to locomote and capture prey. Many Rhizaria synthesize mineral exoskeletons from silica, strontium sulfate, or calcium carbonate. Foraminifera, for instance, typically have reticulopodia, thin, networking pseudopodia, and tests composed of calcium carbonate. Radiolarians, which produce silica exoskeletons, play a large role in the silica cycle. Rhizarians hold significant ecological importance due to their abundance as heterotrophic plankton and their contributions to primary production. Despite being heterotrophs, Rhizarians can harbor photosynthetic algae as symbionts or kleptobionts. Rhizaria are relatively abundant in deep-sea environments, and their presence in hydrothermal vents has been documented.</p>
</section>
<section id="supergorup-stramenopiles" class="level1" data-number="9">
<h1 data-number="9"><span class="header-section-number">9</span> Supergorup: Stramenopiles</h1>
<p>The Stramenopiles are an extremely diverse supergroup, with over 100,000 species classified under it. Stramenopiles are one of the three major taxonomic groupings within the SAR supergroup, along with Alveolata and Rhizaria. This supergroup was previously called Heterokonta, but the name was originally used to indicate that a cell has two dissimilar flagella, due to the wide variety within this group they have been reclassified as Stramenopiles in most modern papers. This group can inhabit almost any environment but is most commonly found in marine habitats. The trophic strategy is also wide-ranging including autotrophic, heterotrophic, and parasitic. Most Stramenopiles are unicellular flagellates and at some point in their lifecycle, they can produce flagellated cells such as gametes or zoospores.</p>
<p>Stramenopiles have several defining characteristics that differentiate them from other supergroups. In most species, Stramenopiles can be distinguished by stiff, tripartite external hairs attached to the flagella. For other species, these hairs can be attached elsewhere on the cellular surface, and in a few species, the flagella hairs have been secondarily lost. The base of the flagellar hari is flexible and inserts into the cell membrane. The second part is dominated by a long stiff tube, the ‘stramen’, and is tipped by many delicate hairs called mastigonemes. The mastigonemes proteins are exclusive to the stramenopile group and are even present in taxa that no longer have hairs, such as diatoms. Many in this group have a form of plastid, called stramenochrome or chromoplast, which allows them to photosynthesize. Two of the most significant autotrophic stramenopiles are brown algae and diatoms, diatoms being one of the most significant primary producers in the ocean ecosystem. Overall, Stramenopiles play a major role in the recycling of carbon on a global scale.</p>
<section id="references" class="level3" data-number="9.0.1">
<h3 data-number="9.0.1" class="anchored" data-anchor-id="references"><span class="header-section-number">9.0.1</span> <strong>References</strong></h3>
<ul>
<li><p>Tomáš Pánek, et al.&nbsp;2016. “First multigene analysis of Archamoebae (Amoebozoa: Conosa) robustly reveals its phylogeny and shows that Entamoebidae represents a deep lineage of the group.” Molecular Phylogenetics and Evolution 98 (pp 41-51).</p></li>
<li><p>Daniel Vaulot, et al.&nbsp;“metaPR2: A database of eukaryotic 18S rRNA metabarcodes with an emphasis on protists.” Molecular Ecology Resources 22, no. 8 (2022): 3188-3201.</p></li>
<li><p>Elena Gerasimova, et al.&nbsp;“Taxonomic Structure of Planktonic Protist Communities in Saline and Hypersaline Continental Waters Revealed by Metabarcoding.” Water 15, no. 11 (2023): 2008.</p></li>
<li><p>Dapeng Xu, et al.&nbsp;“Pigmented microbial eukaryotes fuel the deep sea carbon pool in the tropical Western Pacific Ocean.” Environmental microbiology 20, no. 10 (2018): 3811-3824.</p></li>
<li><p>Dapeng Xu, et al.&nbsp;“Microbial eukaryote diversity and activity in the water column of the South China Sea based on DNA and RNA high throughput sequencing.” Frontiers in microbiology 8 (2017): 1121.</p></li>
<li><p>Archibald, et al.&nbsp;2019. Handbook of the Protists. Cham: Springer International Publishing.</p></li>
<li><p>Ohtsuka, Susumu, et al.&nbsp;2016. Marine protists: Diversity and dynamics. SPRINGER Verlag, JAPAN.</p></li>
<li><p>Lynn, Denis H. 2010. The Ciliated Protozoa. Springer Science. https://doi.org/10.1007/978-1-4020-8239-9.</p></li>
<li><p>Cavalier-Smith T, Chao EE. Phylogeny and evolution of apusomonadida (protozoa: apusozoa): new genera and species. Protist 2010; 161: 549–576.</p></li>
</ul>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>